### DISCUSSION TITLE
Troubleshooting False Alarms in MySQL Exporter: mysql_up Metric Anomaly

### DISCUSSION THREAD

**Discussion by zachary1943** [prometheus/prometheus#13530] [Q&A]
### Proposal

Hello! You described an issue with monitoring multiple MySQL instances using MySQL Exporter. When one instance goes down, other instances also intermittently trigger false alarms. In Prometheus, the mysql_up metric is 0, but the targets under Status appear normal. When checking mysql_up with Curl, it shows 1, and all other metrics are fine. The problem seems to be related to mysql_up being 0, causing false alarms that only disappear when the instance recovers.

**Comment by roidelapluie** [comment]
Thanks for your request. Can you please provide more details like the alerts and expressions that fire?
